embc-app/DataInterfaces/DataInterface.IncidentTask.cs

```diff
@@ -13,7 +13,8 @@ public partial class DataInterface
                 .Include(t => t.Region)
                 .Include(t => t.Community)
                     .ThenInclude(d => d.Region)
-                .Include(t => t.EvacueeRegistrations);
+                .Include(t => t.EvacueeRegistrations)
+                    .ThenInclude(er => er.Evacuees);

```

Comment:
I'm wondering if there's a way to make the count as part of the query so it won't return a huge resultset

Reply:
You may be able to select a tuple that has the object and a count